test/prism.spec.js

```diff
@@ -77,8 +77,8 @@ describe('prismHighlight', () => {
       '}'
     ].join('\n');
 
-    // Use language: 'plain' to simplify the test
-    const result = prismHighlight(input, { tab: '  ', lang: 'plain' });
+    // Use language: 'plain-text' for not supported language with Prism
+    const result = prismHighlight(input, { tab: '  ', lang: 'plain-text' });
```

Comment:
I assume this test case seems to test for not supported language with Prism. But, if we specify `lang: 'plain` the `prismHighlight` function will return `Prism.highlight` result.

So, we want to pass through [return escapeHTML(code);](https://github.com/hexojs/hexo-util/blob/837008b7dd8633aee2484e01d3b85e1e0cf66c26/lib/prism.js#L28) when this test, but current test code pass through [return Prism.highlight(code, Prism.languages[language], language);](https://github.com/hexojs/hexo-util/blob/837008b7dd8633aee2484e01d3b85e1e0cf66c26/lib/prism.js#L24).

I think it happened before, not recently.